sdcard/SYS/CAPTURES/THIS_IS_DEMO_FOLDER.TXT

```diff
@@ -0,0 +1,2 @@
+This is a demo folder. The Capture app now
```

Comment:
This folder used to be SAMPLES, with our sample C16 files.
The reason I rename it, is because it would cost too much memory to hard coded the dir mapping (for Profile switcher func), in which means, the Profile switch func is automatically, without any hard coded dir mapping 

---
Mapping: for example 
/SYS/SAMPLES <---> /CAPTURES

if they don't have same name with different level, it will need hard coded, which is not worth